src/index.ts

```diff
@@ -0,0 +1,29 @@
+import { AuthenticationService } from '@frmscoe/auth-lib';
+import { LoggerService } from '@frmscoe/frms-coe-lib';
+import initializeFastifyClient from './clients/fastify';
+import { configuration } from './config';
+
+export const loggerService: LoggerService = new LoggerService();
+export const authService = new AuthenticationService();
+
```

Comment:
Adding type = intellisense (much nice)
```suggestion
export const authService: AuthenticationService = new AuthenticationService();
```